auth/settings/base.py

```diff
@@ -97,3 +97,4 @@
 
 BRAINTREE_PUBLIC_KEY = ''
 BRAINTREE_PRIVATE_KEY = ''
+BRAINTREE_CRT = '/ssl/api_braintreegateway_com.ca.crt'
```

Comment:
since this looks like a local server path it might be helpful to add a comment here explaining that it is a relative URL